tests/integration/integration_test.go

```diff
@@ -264,7 +262,7 @@ func Test_EventFilters(t *testing.T) {
 			ctx, cancel := context.WithCancel(context.Background())
 			filter, err := flags.PrepareFilter(tc.filterArgs)
 			require.NoError(t, err)
-
+			filter.TraceSelf = true
```

Comment:
Instead of setting this by default, shouldn't we have a cli option to enable this?
Similar to `-t follow` we can have `-t self`.

Reply:
Yea makes sense, I added this.